### PR TITLE
Chore: Fix flaky frontend e2e tests with the explore page

### PR DESCRIPTION
**Pull request by academo**
**What is this feature?**

Fixes a flaky e2e related to the frontend sandbox

**Why do we need this feature?**

To keep the CI pipeline running green

**Who is this feature for?**

Grafana developers

### CODE REVIEW

File: e2e/various-suite/frontend-sandbox-datasource.spec.ts [hunk 1]
```diff
@@ -85,6 +85,8 @@ describe('Datasource sandbox', () => {
         e2e.pages.Explore.visit();
         e2e.components.DataSourcePicker.container().should('be.visible').click();
         cy.contains(DATASOURCE_TYPED_NAME).scrollIntoView().should('be.visible').click();
+        // make sure the datasource was correctly selected and rendered
+        cy.get('span[data-testid="data-testid SandboxDatasourceInstance breadcrumb"]').should('exist');
```
Comment (ashharrison90) [inline e2e/various-suite/frontend-sandbox-datasource.spec.ts]: should be able to do `e2e.components.Breadcrumbs.breadcrumb(DATASOURCE_TYPED_NAME).should('be.visible');` instead everywhere

**Review by ashharrison90 — COMMENTED** [review]
there is also a duplicated version of this test that runs with scenes under the `e2e/scenes` route... does it need the same changes? 🤔

File: e2e/various-suite/frontend-sandbox-datasource.spec.ts (hunk 1 above)
Comment (academo) [reply e2e/various-suite/frontend-sandbox-datasource.spec.ts]: Yes I just changed it

File: e2e/scenes/various-suite/frontend-sandbox-datasource.spec.ts [hunk 2]
```diff
@@ -123,6 +138,10 @@ describe.skip('Datasource sandbox', () => {
         e2e.pages.Explore.visit();
         e2e.components.DataSourcePicker.container().should('be.visible').click();
         cy.contains(DATASOURCE_TYPED_NAME).scrollIntoView().should('be.visible').click();
+        // make sure the datasource was correctly selected and rendered
+        cy.get(`span[data-testid="${selectors.components.Breadcrumbs.breadcrumb(DATASOURCE_TYPED_NAME)}"]`).should(
+          'be.visible'
+        );
```
Comment (ashharrison90) [inline e2e/scenes/various-suite/frontend-sandbox-datasource.spec.ts:144]: you can simplify this further i think
```suggestion
        // make sure the datasource was correctly selected and rendered
        e2e.components.Breadcrumbs.breadcrumb(DATASOURCE_TYPED_NAME).should(
          'be.visible'
        );
```

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍

**Pull request merged**